wlauto/common/android/workload.py

```diff
@@ -127,6 +127,11 @@ class ApkWorkload(Workload):
     :view: The class of the main view pane of the app. This needs to be defined in order
            to collect SurfaceFlinger-derived statistics (such as FPS) for the app, but
            may otherwise be left as ``None``.
+    :launch_main: If ``False``, the default activity will not be launched (during setup),
```

Comment:
We need the WA chaps to check if they like this.  See my comment in __init__.py as to why you actually require it.